<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sekhar-PC" name="IETest" tests="2" failures="1" timestamp="22 Nov 2019 07:06:58 GMT" time="52.527" errors="0">
  <testcase name="step1" time="0.029" classname="TestCases.TC1"/>
  <testcase name="step1" time="0.001" classname="TestCases.TC2">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at TestCases.TC2.step1(TC2.java:19)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- step1 -->
</testsuite> <!-- IETest -->
